src/test/java/com/exasol/adapter/json/RequestJsonParserTest.java

```diff
@@ -117,85 +130,37 @@ public void testParseSetPropertiesRequest() throws Exception {
-        "                    [" +
-        "                        {" +
-        "                            \"dataType\" :" +
-        "                            {" +
-        "                                \"precision\" : 18," +
-        "                                \"scale\" : 0," +
-        "                                \"type\" : \"DECIMAL\"" +
-        "                            }," +
-        "                            \"name\" : \"ID\"" +
-        "                        }" +
-        "                    ]," +
-        "                    \"name\" : \"T1\"" +
-        "                }," +
-        "                {" +
-        "                    \"columns\" :" +
-        "                    [" +
-        "                        {" +
-        "                            \"dataType\" :" +
-        "                            {" +
-        "                                \"precision\" : 18," +
-        "                                \"scale\" : 0," +
-        "                                \"type\" : \"DECIMAL\"" +
-        "                            }," +
-        "                            \"name\" : \"ID\"" +
-        "                        }" +
-        "                    ]," +
-        "                    \"name\" : \"T2\"" +
-        "                }" +
-        "            ]," +
-        "    \"pushdownRequest\" : " +
-        "    {" +
-        "        \"type\" : \"select\"," +
-        "        \"from\" : " +
-        "        {" +
-        "            \"type\": \"join\"," +
-        "            \"join_type\": \"inner\"," +
-        "            \"left\":" +
-        "            {" +
-        "                \"name\" : \"T1\"," +
-        "                \"type\" : \"table\"" +
-        "            }," +
-        "            \"right\":" +
-        "            {" +
-        "                \"name\" : \"T2\"," +
-        "                \"type\" : \"table\"" +
-        "            }," +
-        "            \"condition\":" +
-        "            {" +
-        "                \"left\" :" +
-        "                {" +
-        "                        \"columnNr\" : 0," +
-        "                        \"name\" : \"ID\"," +
-        "                        \"tableName\" : \"T1\"," +
-        "                        \"type\" : \"column\"" +
-        "                }," +
-        "                \"right\" :" +
-        "                {" +
-        "                        \"columnNr\" : 0," +
-        "                        \"name\" : \"ID\"," +
-        "                        \"tableName\" : \"T2\"," +
-        "                        \"type\" : \"column\"" +
-        "                }," +
-        "                \"type\" : \"predicate_equal\"" +
-        "            }" +
-        "        }" +
-        "    }," +
-        "    \"type\" : \"pushdown\"" +
-        "}";
+    void testSimpleInnerJoinRequest() throws Exception {
```

Comment:
Move "+" in front. Terminate each line with ` //` to prevent formatter from scrambling the lines.

Reply:
done